src/client/components/pages/RoomAdmin/RoomAdmin.tsx

```diff
@@ -292,7 +292,7 @@ const RoomAdmin: FunctionComponent = (): ReactElement => {
                         name="room-filter"
                         placeholder="Filter by Room Name"
                         value={roomFilter}
-                        label="Change to filter the list of meetings by room"
+                        label="Change to filter the list of rooms by room"
```

Comment:
It probably makes more sense to have the text "Change to filter the list of rooms by room **name**" here, since the campus and building are also part of the room information in general.